### DISCUSSION TITLE
How to "Run make bootstrap" & "cd into the project repo " in short how do I use Kickstarter source code

### DISCUSSION THREAD

**Discussion by lalitsharmaking** [jesseduffield/lazygit#2008] [Q&A]
encountered this google service missing error when I try to run the app Kickstarter from GitHub in Android Studio.

Tried to look for answers and I found that I am supposed to run "make bootstrap" but I really can't quite understand how you do it (i'm still a novice in coding/android/github in general).

Here is the error:

File google-services.json is missing. The Google Services Plugin cannot function without it. Searched Location: C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\external\min21\debug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\externalMin21\debug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\debug\externalMin21\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\external\debug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\external\min21\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\external\min21Debug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\externalMin21\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\debug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\externalMin21Debug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\external\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\src\externalDebug\google-services.json C:\Users\naufa\AndroidStudioProjects\android-oss\app\google-services.json

It showed this after I cloned it using git bash 
But Before it showed ( when I downloaded zip) - line missing/ can't run app into this directory 

The scenario was:

I imported the project into Android Studio, when I try to run the app, the error above pops up. I browsed for solutions and found out that I need to run "make bootstrap" in some sort of terminal/CMD, but I have zero ideas on how to do that, heck i don't even know what bootstrap is.

There was a lot of different answers that i have no idea how to do since the answers was meant for non-beginners. I'm so confused, some help would be greatly appreciated.

I'm running Windows 11 64-bit, if there is any other info I should provide please do tell me. Here's the Clone link if anyone would like to open it in Android Studio = https://github.com/kickstarter/android-oss.git

I am completely new in this & I really can't understand anything so it would be very helpful if someone can explain in simple words to me

**Comment by mark2185** [comment]
This has nothing to do with `lazygit` (a command line tool for interacting with `git` repositories) you should ask the question on their discussions page.

**Reply by lalitsharmaking** [reply]
Brother I don't know much about how these things work I have recently started coding & using GitHub

**Reply by mark2185** [reply]
`jesseduffield`'s project `lazygit` and `kickstarter`'s project `android-oss` are two different projects on `github`, and what you are trying to run has nothing to do with `lazygit`.

I see you have opened [an issue](https://github.com/kickstarter/android-oss/issues/1647) on the `android-oss` project, that's what I meant you should do. I replied there, and we will try to debug it there.
